capstone-server/src/main/java/com/hcl/capstoneserver/user/entities/Supplier.java

```diff
@@ -1,33 +1,21 @@
 package com.hcl.capstoneserver.user.entities;
 
-import com.hcl.capstoneserver.user.UserRole;
+import com.hcl.capstoneserver.user.UserType;
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
 
 @Entity
-public class Supplier extends AppUser{
+public class Supplier extends Person {
     @Column(unique = true)
     private int supplierId;
 
-    private String name;
-    private String address;
-    private String email;
-    private String phone;
-    private Float interestRate;
-
     public Supplier() {
     }
 
-    public Supplier(String userId, String password, int supplierId,
-                    String name, String address, String email, String phone, Float interestRate) {
-        super(userId, password, "SUPPLIER");
+    public Supplier(String userId, String password, UserType userType, String name, String address, String email, String phone, Float interestRate, int supplierId) {
```

Comment:
```suggestion
    public Supplier(String userId, String password, String name, String address, String email, String phone, Float interestRate, int supplierId) {
```

Supplier userType is a constant. it should not be changed from outside